filebeat/input/winlog/input.go

```diff
@@ -135,16 +135,24 @@ runLoop:
 		for cancelCtx.Err() == nil {
 			records, err := api.Read()
 			if eventlog.IsRecoverable(err) {
-				log.Errorw("Encountered recoverable error when reading from Windows Event Log", "error", err)
+				log.Errorw("Encountered recoverable error when reading from Windows Event Log", "error", err, "channel", api.Channel())
```

Comment:
On L89 it appears that the logger is already setup to include the `channel` value. So there is no need to add `channel`.